app/models/collection.rb

```diff
@@ -3,7 +3,15 @@
 # Web archive collection
 class Collection < ApplicationRecord
   self.primary_key = 'druid'
-  validates_presence_of :title, :druid, :active, :embargo_months
+  validates_presence_of :title, :druid, :active, :embargo_months, :wasapi_provider_account
```

Comment:
Should this be validating `wasapi_provider` and `wasapi_account`?

Reply:
At least from the UI perspective, only `wasapi_provider_account` is needed (not `wasapi_provider` and `wasapi_account` separately).